.github/CODEOWNERS

```diff
@@ -1,2 +1,3 @@
 # Maintained by the API experience team
-*       @mongodb/apix
+mongodbatlas/*       @mongodb/apix
```

Comment:
the new version of the client will use a different notification mechanism. 
This will keep the current notification mechanism for v0 and stop unneeded noise for changes driven by generated client.

Reply:
this leaves out changes to the go.mod/go.sum files, so we won't get notified on dependabot changes, same with the gh actions folders

Reply:
I was reading the docs for codeowners and seems like the not(!) operator is not supported :(